### ISSUE TITLE
WLASL_v0.3.json file is missing some videos information

### ISSUE THREAD

**Issue by SalsabilRefaei** [dxli94/WLASL#69]
I filled the form provided to request for the missing videos. Thankfully, I got all the videos which where 21095 videos. when I checked the provided info in the JSON file it include only 21083 videos information. it may be 12 video only but I will appreciate it if I got the complete JSON file 

![image](https://github.com/dxli94/WLASL/assets/139865772/5b7d2bb9-3507-4982-9115-473d6917ff61)

![image](https://github.com/dxli94/WLASL/assets/139865772/727b3ac4-70b0-410e-a60a-609ef2fbca53)

**Comment by HakeemYB** [comment]
are you able to train this model for 30-40 classes?

**Comment by XiongTLu** [comment]
Yes, I can. And I can use my pretrained model in all classes of WLASL.---- Replied Message ----FromHakeem Yar ***@***.***>Date4/1/2024 ***@***.***>***@***.***>SubjectRe: [dxli94/WLASL] WLASL_v0.3.json file is missing some videos information (Issue #69)are you able to train this model for 30-40 classes?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by XiongTLu** [comment]
Congratulation,  you can got missing videos. But I'm not face this problems like you, maybe you can check you JSON file and compare with author's JSON file.---- Replied Message ----FromSalsabil ***@***.***>Date3/29/2024 ***@***.***>***@***.***>Subject[dxli94/WLASL] WLASL_v0.3.json file is missing some videos information (Issue #69)I filled the form provided to request for the missing videos. Thankfully, I got all the videos which where 21095 videos. when I checked the provided info in the JSON file it include only 21083 videos information. it may be 12 video only but I will appreciate it if I got the complete JSON fileimage.png (view on web)image.png (view on web)—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by HakeemYB** [comment]
> Yes, I can. And I can use my pretrained model in all classes of WLASL.---- Replied Message ----FromHakeem Yar ***@***.***>Date4/1/2024 ***@***.***>***@***.***>SubjectRe: [dxli94/WLASL] WLASL_v0.3.json file is missing some videos information (Issue #69)are you able to train this model for 30-40 classes?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

How can we train for 30 classes can you help?

**Comment by XiongTLu** [comment]
I'am sorry, I never try to train in 30 classes. Maybe I can't help you about this problem. But I still focus on Sign Language Recognition. If got any idea can help you, I will send email to you. Hoping you get great result.
---- Replied Message ----
FromHakeem Yar ***@***.***>Date4/5/2024 ***@***.***>***@***.***>,
***@***.***>SubjectRe: [dxli94/WLASL] WLASL_v0.3.json file is missing some videos information (Issue #69)
Yes, I can. And I can use my pretrained model in all classes of WLASL.---- Replied Message ----FromHakeem Yar @.>Date4/1/2024 @.>@.>SubjectRe: [dxli94/WLASL] WLASL_v0.3.json file is missing some videos information (Issue #69)are you able to train this model for 30-40 classes?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: @.>
How can we train for 30 classes can you help?

**Comment by yanxiyue-cloud** [comment]
@SalsabilRefaei Hello, may I ask if you could share the complete video? I will be extremely grateful.
